db/models/applications.js

```diff
@@ -96,6 +106,9 @@ class PendingApplication extends Model {
             }
         });
     }
+    static countCompleted(applications) {
+        return applications.filter(app => app.currentProgressState === 'COMPLETE').length;
```

Comment:
replaced lodash `countBy` with JS `filter` method as `countBy` stopped working after I change the type of `currentProgressState` to `ENUM` (flaky + weird?).

Should we stay away from ENUM or go ahead with this ?

Reply:
Hmm weird – it's probably fine to leave this as you've got it here, then 👍